src/utils/featheryClient/index.ts

```diff
@@ -22,83 +22,60 @@ import { initializeIntegrations } from '../../integrations/utils';
-  staging: 'https://staging.feathery.io/api/',
-  production: 'https://api.feathery.io/api/',
-  productionAU: 'https://api-au.feathery.io/api/',
-  productionEU: 'https://api-eu.feathery.io/api/',
-  productionCA: 'https://api-ca.feathery.io/api/'
-};
-
-const CDN_URL_OPTIONS = {
-  local: 'http://localhost:8006/api/',
-  staging: 'https://staging.feathery.io/api/',
-  production: 'https://cdn.feathery.io/api/',
-  productionAU: 'https://cdn-au.feathery.io/api/',
-  productionEU: 'https://cdn-eu.feathery.io/api/',
-  productionCA: 'https://cdn-ca.feathery.io/api/'
-};
-
-const STATIC_URL_OPTIONS = {
-  local: 'http://localhost:8006/api/',
-  staging: 'https://staging.feathery.io/api/',
-  production: 'https://api-static-2.feathery.io/api/',
-  productionAU: 'https://api-au.feathery.io/api/',
-  productionEU: 'https://api-eu.feathery.io/api/',
-  productionCA: 'https://api-ca.feathery.io/api/'
-};
-
-const S3_URL_OPTIONS = {
-  local: 'http://localhost:8006',
-  staging: 's3.us-west-1.amazonaws.com',
-  production: 's3.us-west-1.amazonaws.com',
-  productionAU: 's3.ap-southeast-2.amazonaws.com',
-  productionEU: 's3.eu-west-1.amazonaws.com',
-  productionCA: 's3.ca-central-1.amazonaws.com'
-};
-
-type URL_ENUM = keyof typeof API_URL_OPTIONS;
-let environment: URL_ENUM = 'production';
+setEnvironment('production');
 try {
-  environment = (process.env.BACKEND_ENV || 'production') as URL_ENUM;
+  setEnvironment((process.env.BACKEND_ENV || 'production') as URL_ENUM);
 } catch (e) {} // process.env won't exist in production build
 
-export let API_URL = API_URL_OPTIONS[environment];
-export let CDN_URL = CDN_URL_OPTIONS[environment];
-export let STATIC_URL = STATIC_URL_OPTIONS[environment];
-export let S3_URL = S3_URL_OPTIONS[environment];
+export let API_URL = getApiUrl();
+export let CDN_URL = getCdnUrl();
+export let STATIC_URL = getStaticUrl();
+export let S3_URL = getS3Url();
 
 export const updateRegionApiUrls = (region: string) => {
   if (region === 'au') {
-    CDN_URL = CDN_URL_OPTIONS.productionAU;
-    API_URL = API_URL_OPTIONS.productionAU;
-    STATIC_URL = STATIC_URL_OPTIONS.productionAU;
-    S3_URL = S3_URL_OPTIONS.productionAU;
+    setEnvironment('productionAU');
+    CDN_URL = getCdnUrl();
+    API_URL = getApiUrl();
+    STATIC_URL = getStaticUrl();
+    S3_URL = getS3Url();
   } else if (region === 'eu') {
-    CDN_URL = CDN_URL_OPTIONS.productionEU;
-    API_URL = API_URL_OPTIONS.productionEU;
-    STATIC_URL = STATIC_URL_OPTIONS.productionEU;
-    S3_URL = S3_URL_OPTIONS.productionEU;
+    setEnvironment('productionEU');
+    CDN_URL = getCdnUrl();
+    API_URL = getApiUrl();
+    STATIC_URL = getStaticUrl();
+    S3_URL = getS3Url();
   } else if (region === 'ca') {
-    CDN_URL = CDN_URL_OPTIONS.productionCA;
-    API_URL = API_URL_OPTIONS.productionCA;
-    STATIC_URL = STATIC_URL_OPTIONS.productionCA;
-    S3_URL = S3_URL_OPTIONS.productionCA;
+    setEnvironment('productionCA');
+    CDN_URL = getCdnUrl();
```

Comment:
all of this is unnecessarily duplicated code between the if branches. you should just do something like:

const environmentMap = {ca: 'productionCA', eu: 'productionEU', ...}

and then:
setEnvironment(environmentMap[region]);
CDN_URL = getCdnUrl();
...